--- origsrc/gnome-games-2.16.3/configure.in	2007-01-28 10:12:51.000000000 -0600
+++ src/gnome-games-2.16.3/configure.in	2007-08-17 14:23:04.937500000 -0500
@@ -202,7 +202,7 @@
 AC_SUBST(ESD_CFLAGS)
 AC_SUBST(ESD_LIBS)
 
-GETTEXT_PACKAGE=gnome-games
+GETTEXT_PACKAGE=gataxx
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [The gettext package name])
 AM_GLIB_GNU_GETTEXT
@@ -218,6 +218,11 @@
 AC_C_CONST
 AC_C_INLINE
 
+# check for getaddrinfo (libgames-support/games-network.c)
+AC_SEARCH_LIBS(getaddrinfo, getaddrinfo, [AC_CHECK_HEADERS(getaddrinfo.h)],
+  [AC_MSG_ERROR([Functions getaddrinfo and gai_strerror are required])
+])
+
 if test "x${prefix}" = "xNONE"; then
    AC_DEFINE_UNQUOTED(GNOME_ICONDIR, "${ac_default_prefix}/share/pixmaps", [Pix
 map directory])
--- origsrc/gnome-games-2.16.3/libgames-support/games-network.c	2007-01-28 09:40:38.000000000 -0600
+++ src/gnome-games-2.16.3/libgames-support/games-network.c	2007-08-17 13:29:42.187500000 -0500
@@ -41,6 +41,9 @@
 #include <errno.h>
 #include <netdb.h>
 
+#ifdef HAVE_GETADDRINFO_H
+#include <getaddrinfo.h>
+#endif
 
 #if defined (HAVE_HOWL)
 /* Workaround broken howl including config.h */
